src/EnergyPlus/OutputProcessor.cc

```diff
@@ -867,8 +867,7 @@ namespace OutputProcessor {
                                             ipsc->cAlphaFieldNames(fldIndex + 1),
                                             ipsc->cAlphaArgs(fldIndex + 1)));
                     ShowContinueError(state, "...will not be shown with the Meter results.");
-                    foundBadSrc = true;
```

Comment:
One line fix. A missing component variable/meter no longer sets an error condition that prevents the custom meter from being created.

Reply:
Looks good to me!